src/schema/nodes/affiliations.ts

```diff
@@ -13,51 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { NodeSpec } from 'prosemirror-model'
 
-import { schema } from '../index'
-import { ManuscriptNode } from '../types'
+import { NodeSpec } from 'prosemirror-model'
 
-interface Attrs {
-  id: string
-}
-
-export interface AffiliationsNode extends ManuscriptNode {
-  attrs: Attrs
-}
+// This node has no representation in json-schema
+// It exists for the purpose of styling in the UI
 
 export const affiliations: NodeSpec = {
-  content: 'section_title? affiliation*',
+  content: 'affiliation*',
```

Comment:
I remember this caused an issue on the editor, did you try to upload something to the editor? 
https://github.com/Atypon-OpenSource/manuscripts-transform/pull/50#issuecomment-1810880054

Reply:
Yes, it works. I'm guessing the fact it was a section before required it to have a title.

Reply:
Oh.. yeah. Thanks.